beacon_node/beacon_chain/src/validator_monitor.rs

```diff
@@ -423,6 +473,33 @@ impl<T: EthSpec> ValidatorMonitor<T> {
                         );
                     }
                 }
+
+                // Indicates the number of sync committee signatures that made it into
+                // a sync aggregate.
+                if let Some(sync_committee) = &self.current_sync_committee {
+                    if sync_committee.contains(pubkey) {
+                        let epoch_summary = monitored_validator.summaries.read();
```

Comment:
This is incorrect as the sync committee for the current state may not be equal to sync committee at `prev_epoch`. 
Will probably need to plumb sync_committee corresponding to `prev_epoch` through the `EpochProcessingSummary`  object.

Reply:
Yeah I think that would be a better approach. IMO we should get rid of the cached sync committee and avoid doing anything sync-related in `process_valid_state`.

I'll rebase the `altair-vc` branch so it brings in @paulhauner's latest changes to the validator monitor. I think we could stash the `current_sync_committee` at this point in epoch processing: https://github.com/sigp/lighthouse/blob/6e3ca48cb934a63cafdc940068825a48cba740d1/consensus/state_processing/src/per_epoch_processing/altair.rs#L32-L33

Reply:
I think we could also report on sync inclusions for the current epoch (`epoch - 1`), as unlike attestations can't be included after a whole epoch lag

Reply:
> I think we could also report on sync inclusions for the current epoch `(epoch - 1)`

Yeah I think this would make things easier. Otherwise, we would need to get the `sync_committee` at (`epoch - 2`) which wouldn't be easy on sync_committee transition epochs.

Reply:
Fixed in bd6a51b